trustchain-core/src/utils.rs

```diff
@@ -99,6 +100,15 @@ pub fn get_did_suffix(did: &str) -> &str {
     did.split(':').last().unwrap()
 }
 
+/// Converts a short-form DID into a complete DID.
+pub fn get_did_from_suffix(did_suffix: &str, method: &str) -> String {
```

Comment:
is method `ion:test` for testnet? Do we need an optional network instead to distinguish as in the `ION` type?